common/dns.go

```diff
@@ -6,12 +6,13 @@ import (
 	"net"
 	"time"
 
-	"github.com/giantswarm/cluster-test-suites/internal/state"
 	"github.com/giantswarm/clustertest/pkg/application"
 	"github.com/giantswarm/clustertest/pkg/logger"
 	"github.com/giantswarm/clustertest/pkg/wait"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	"github.com/giantswarm/cluster-test-suites/internal/state"
```

Comment:
This is aligned with [what we described here](https://github.com/giantswarm/fmt/blob/master/go/imports.md).